public/ng-views/maps.html

```diff
@@ -681,11 +842,21 @@ <h4 ng-click="combosView=!combosView">
         <span class="badge" ng-show="!scenarioCompare" ng-click="scenarioCompare=true">
           Toggle Compare <i class="fa fa-toggle-off"></i>
         </span>
+        <span class="badge pull-right" ng-click="saveCombo=!saveCombo">
+          <i class="fa fa-cloud-upload"> </i> Create Scenario
+        </span>
+
+      </span>
+        <span ng-show="scenarioCompare">
+        Click right <i class="fa fa-circle-o"></i> to compare. Leave all blank for baseline.
       </span>
-    </h5>
+      <span ng-show="!scenarioCompare">
+        Select a scenario to perform a single point analysis.
+      </span>
+    </h6>
 
     <table class="table table-condensed table-hover">
-      <thead>
+      <thead style="font-size: .8em">
         <tr>
           <th><i class="fa fa-check-square"></i></th>
           <th>Name</th>
```

Comment:
This whole section is critically broken. 

![image](https://cloud.githubusercontent.com/assets/6053396/8777731/d412fc44-2ec7-11e5-8ce9-ffa0d7dcb0c5.png)

As you can see, there is no scroll capacity so you can't even access the scenarios that are created. Please refer back to your original pull request in early June where I brought this up. This breaks the functionality of the app and is a serious problem.